home/ivp-planning/user-interface/the-top-bar.md

```diff
@@ -34,10 +35,19 @@ The text menu contains four submenus:
 
 
 * Settings:
-  * _**Snapping:**_ toggles the [snapping ](../machines/snapping.md)of machines and objects in the [2D-View](the-2d-panel.md)
   * _**Language:**_ [sets the language](../getting-started/language-options.md) of the software
   * _**Settings:**_ opens the [settings panel](settings-panel.md)
-  * _**License:**_ opens the [license manager panel](license-manager-panel.md)\
+  * _**License:**_ opens the [license manager panel](license-manager-panel.md)
+
+* Windows:
+  * _**Layout:**_ Opens a small submenu that lets you choose a [window layout](layouts.md)
+  * _**Hierarchy:**_ opens the [hierarchy panel](the-machine-list.md), opened and docked by default
+  * _**Library:**_ opens the [library panel](machine-database-panel.md), opened and docked by default
+  * _**Properties:**_ opens the [properties panel](the-info-panel.md), opened and docked by default
+  * _**Camera List:**_ opens the [camera list panel](license-manager-panel.md)
+  * _**Video:**_ opens the [video panel](license-manager-panel.md)
+  * _**Settings:**_ opens the [settings panel](license-manager-panel.md)
+  * _**Markups:**_ opens the [Markups panel](../advanced-tools/markup-tool.md)
```

Comment:
This page does not exist yet, we should remove the link until the page is published